whittle/prunning/layerwrapper.py

```diff
@@ -0,0 +1,35 @@
+import torch
+import torch.nn as nn
+
+
+class WrappedGPT:
+    """
+    This class wraps a GPT layer for specific operations.
```

Comment:
Also maybe it makes sense to move this to the `modules` folder?

Reply:
This class is only there for the sake of wanda prunning the difference is that the wrapped layer will have a scaler factor that will be used in prunning. I think since it is only used for wanda it should be kept in the prunning folder. What do you think?